src/meltano/cli/invoke.py

```diff
@@ -105,13 +105,16 @@ def invoke(
         plugin = plugins_service.find_plugin(
             plugin_name, plugin_type=plugin_type, invokable=True
         )
+        tracker.add_contexts(PluginsTrackingContext([(plugin, command_name)]))
+        tracker.track_command_event(cli_tracking.STARTED)
     except PluginNotFoundError as err:
-        with tracker.with_contexts(cmd_ctx):
-            tracker.track_command_event(cli_tracking.ABORTED)
+        tracker.track_command_event(cli_tracking.STARTED)
+        tracker.track_command_event(cli_tracking.ABORTED)
         raise err
```

Comment:
```suggestion
        raise
```

Using a blank raise will re-raise the exception that is currently being handled.